apps/backend/gateway/src/main/resources/application-dev.yml

```
@@ -0,0 +1,6 @@
+spring:
+  jpa:
+    show-sql: true
+  sql:
+    init:
+      schema-locations: classpath:schema.sql
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)